kani-compiler/src/codegen_cprover_gotoc/codegen/place.rs

```diff
@@ -752,6 +780,10 @@ impl<'tcx> GotocCtx<'tcx> {
     }
 }
 
+fn is_box(ty: Ty) -> bool {
+    matches!(ty.kind(), TyKind::RigidTy(RigidTy::Adt(def, _)) if def.is_box())
+}
+
```

Comment:
Shouldn't these wrappers live in the compiler?

Reply:
Not saying "don't add them here" just that we should eventually migrate them.

Reply:
Yes, they should and will. It's just not high priority at this point. We're trying to focus on adding information that cannot be retrieved with stable APIs at all.